### PR TITLE
Correct Annex list

### PR DESCRIPTION
**Pull request by zvr**
A number of informative documents were copied from being annexes of the spec to their own repository https://github.com/spdx/using.

This PR removes them from the spec repo, and renumbers the rest of the annexes.
It also updates the mkdocs configuration to use this new list.

Signed-off-by: Alexios Zavras <github@zvr.gr>

### CODE REVIEW

**Comment by rnjudge** [comment]
Discussed in July 16th tech call. LGTM.

**Review by goneall — APPROVED** [review]
LGTM

**Comment by goneall** [comment]
@zvr - can you resolve the merge conflicts - then I will merge based on the comments above

File: mkdocs.yml [hunk 1]
```diff
@@ -328,17 +328,12 @@ nav:
...
-  - 'RDF Object Model and Identifier Syntax': annexes/RDF-object-model-and-identifier-syntax.md
-  - 'SPDX License Expressions': annexes/SPDX-license-expressions.md
-  - 'SPDX license list matching guidelines': annexes/license-matching-guidelines-and-templates.md
-  - 'Using SPDX short identifiers in Source Files': annexes/using-SPDX-short-identifiers-in-source-files.md
-  - 'Using SPDX to comply with norms, standards and regulation': annexes/using-SPDX-to-comply-with-industry-guidance.md
-  - 'Including Security Information in SPDX': annexes/including-security-information-in-SPDX.md
-  - 'SPDX Lite': annexes/SPDX-Lite.md
-  - 'Cross-referencing in SPDX 3': annexes/cross-reference.md
-  - 'Package URL specification': annexes/pkg-url-specification.md
+  - 'A: Diffs from Previous Editions': annexes/diffs-from-previous-editions.md
```
Comment (bact) [inline mkdocs.yml:331]: 'A: Diffs' -> 'A: Differences'

**Review by bact — CHANGES_REQUESTED** [review]
Use full word

(same as in [v2.3](https://spdx.github.io/spdx-spec/v2.3/diffs-from-previous-editions/))

File: mkdocs.yml [hunk 2]
```diff
@@ -328,17 +328,12 @@ nav:
...
+  - 'B: RDF Object Model and Identifier Syntax': annexes/RDF-object-model-and-identifier-syntax.md
+  - 'C: SPDX License Expressions': annexes/SPDX-license-expressions.md
+  - 'D: SPDX license list matching guidelines': annexes/license-matching-guidelines-and-templates.md
+  - 'E: Package URL specification': annexes/pkg-url-specification.md
```
Comment (bact) [inline mkdocs.yml:335]: Some annex headings capitalize the first word and lowercase the rest, while the other capitalize (almost) every words. Should stick to one style.

(Exceptions can be made for proper nouns/names)

**Review by bact — CHANGES_REQUESTED** [review]

File: mkdocs.yml (hunk 2 above)
Comment (bact) [reply mkdocs.yml:335]: I'm taking the titles from the Annex files themselves:

```
  - 'A: Differences from previous editions': annexes/diffs-from-previous-editions.md
  - 'B: RDF object model and identifier syntax': annexes/RDF-object-model-and-identifier-syntax.md
  - 'C: SPDX license expressions': annexes/SPDX-license-expressions.md
  - 'D: License matching guidelines and templates': annexes/license-matching-guidelines-and-templates.md
  - 'E: Package URL specification v1': annexes/pkg-url-specification.md
```

**Comment by bact** [comment]
@bobmartin3000 what is the prefer capitalization style for title and section names please?

We have quite a mix currently. For example:

- [2.1 Normative References](https://github.com/spdx/spdx-spec/pull/1006/files)
- [3 Terms and Definitions](https://github.com/spdx/spdx-spec/pull/1004/files)
- [Annex A: Differences from previous editions (Informative)](https://github.com/spdx/spdx-spec/blob/development/v3.0.1/docs/annexes/diffs-from-previous-editions.md)
- [Annex B: RDF object model and identifier syntax (Normative)](https://github.com/spdx/spdx-spec/blob/development/v3.0.1/docs/annexes/RDF-object-model-and-identifier-syntax.md)
- [Annex E: Package URL specification v1 (Normative)](https://github.com/spdx/spdx-spec/blob/development/v3.0.1/docs/annexes/pkg-url-specification.md)
- [Annex F: History, Motivation and Rational (Informative)](https://github.com/spdx/spdx-spec/pull/1008/files)

**Comment by bobmartin3000** [comment]
I would capitalize main words like the first two and last one

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Arthit Suriyawongkul ***@***.***>
Sent: Monday, August 5, 2024 6:04:05 AM
To: spdx/spdx-spec ***@***.***>
Cc: Robert A Martin ***@***.***>; Mention ***@***.***>
Subject: [EXT] Re: [spdx/spdx-spec] Correct Annex list (PR #992)

@bobmartin3000 what is the prefer capitalization style for title and section names please? We have quite a mix currently. For example: 2. 1 Normative References 3 Terms and Definitions Annex A: Differences from previous editions (Informative)


@bobmartin3000<https://github.com/bobmartin3000> what is the prefer capitalization style for title and section names please?

We have quite a mix currently. For example:

  *   2.1 Normative References<https://github.com/spdx/spdx-spec/pull/1006/files>
  *   3 Terms and Definitions<https://github.com/spdx/spdx-spec/pull/1004/files>
  *   Annex A: Differences from previous editions (Informative)<https://github.com/spdx/spdx-spec/blob/development/v3.0.1/docs/annexes/diffs-from-previous-editions.md>
  *   Annex B: RDF object model and identifier syntax (Normative)<https://github.com/spdx/spdx-spec/blob/development/v3.0.1/docs/annexes/RDF-object-model-and-identifier-syntax.md>
  *   Annex E: Package URL specification v1 (Normative)<https://github.com/spdx/spdx-spec/blob/development/v3.0.1/docs/annexes/pkg-url-specification.md>
  *   Annex F: History, Motivation and Rational (Informative)<https://github.com/spdx/spdx-spec/pull/1008/files>

**Comment by bact** [comment]
> I would capitalize main words like the first two and last one Get Outlook for iOS<https://aka.ms/o0ukef>

Thank you @bobmartin3000. Then if everyone agree, I will update headings in mkdocs.yml and in actual files (chapters/sections/annexes) to consistently match that style.

**Comment by kestewart** [comment]
@zvr, @bact - looks like we've got some conflicts now that need to be resolved -   can you take a pass at sorting them?

**Review by kestewart — APPROVED** [review]

**Comment by goneall** [comment]
@zvr - Could you resolve the merge conflicts?  It looks like most of them are referencing deleted files, so it shouldn't be too challenging.

We put in a separate issue for the Title casing #1041 - so we can merge this and fix the titles later.  Once the merge conflicts are resolved, we can merge.

**Comment by goneall** [comment]
This will be replaced by a new PR on 11 August

**Pull request closed without merging**